index.js

```diff
@@ -1608,6 +1608,18 @@
     return Bifunctor.methods.bimap(bifunctor)(f, g);
   }
 
+  //# mapLeft :: Bifunctor f => (a -> b, f a c) -> f b c
+  //.
+  //. Maps the given function over the first type argument of a Bifunctor
```

Comment:
We usually describe functions at the value level. It's not clear how to describe this function without referencing the type level. Any ideas?

Also, this sentence is missing its full stop. ;)

Reply:
> Any ideas?

None. That's how I ended up referencing "type argument". Fantasy Land sidesteps any mention of values.

Reply:
"Maps the given function over the left side of a Bifunctor."
?

Reply:
> "Maps the given function over the left side of a Bifunctor."

:thumbsup: